.github/ISSUE_TEMPLATE/1 new type.md

```diff
@@ -0,0 +1,6 @@
+---
+name: 💡 Suggest new type
+about: __________________
+title: 'Proposal: '
+labels: 'type addition'
```

Comment:
This should also use a "issue form" to ask to search existing requests and types first.